### ISSUE TITLE
Scrollable style does not work in firefox 59.0.2 - windows 10

### ISSUE THREAD

**Issue by ruperaz** [akveo/nebular#347]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [ x] bug report
* [ ] feature request

### Issue description

**Current behavior:**

 The scrollable style does not work in firefox 59.0.2 - windows 10

![image](https://user-images.githubusercontent.com/4202458/38326706-b729a2b8-385b-11e8-909d-e12c20ea6d83.png)

**Expected behavior:**

**Steps to reproduce:**

visit the provided demo link at https://akveo.github.io/nebular in firefox and windows 10

**Comment by nnixaa** [comment]
Hi @ruperaz, as far as I can tell, there is no way to change firefox scrollbars with a use of CSS. Currently, we are not planning on adding JS-driven solution to make scrollbars styled in firefox.
Leaving this issue opened to see if there is any interest in this feature.

**Comment by Prefix1802** [comment]
@nnixaa 

As i am now well aware of Firefox not implementing/supporting webkit-scrollbar in CSS. I am having a really hard time understanding why the team does not want to provide supprt via a JS-driven solutions.

I hope you can understand when i say this, and please don't take it personally when i say you ;)

You can not expect your community to all disregard Firefox and use Chrome. My guess is that its not too much of work to provide support for this issue.

Now, only because i can not present an app that behaves like an app, for my firefox or mobile phone users, i am really asking myself if i want to go the Nebular way.

BlurAdmin does an amazing job for me already and has custom scrollbars.

P.S. Don't get me wrong. I am very fond of Nebular and the work you all are doing.

**Comment by nnixaa** [comment]
Hey @Prefix1802, I'm pretty sure that the decision made by Firefox team has a reason for that.
Moreover, I'm not sure I understand how the "not styled" sidebars are affecting Firefox users? As in my view, "not styled" sidebars would be a default behavior which means it is expected by Firefox users. 

Could you please elaborate, probably I'm missing the point.

**Comment by Prefix1802** [comment]
@nnixaa 

By this:

"I am having a really hard time understanding why the team does not want to provide supprt via a JS-driven solution."

I meant the nebular team.

"I'm not sure I understand how the "not styled" sidebars are affecting Firefox users?"

![afbeelding](https://user-images.githubusercontent.com/40199655/44354338-dda7f380-a4a9-11e8-9fee-51898520f412.png)

**Comment by nnixaa** [comment]
@Prefix1802, yeah, I understand that you mentioned our team, just referenced the Firefox team.
As for the screenshot - you can simply adjust the `compacted` width of the sidebar by using `sidebar-width-compact` nb-theme variable. This way you can make it wider avoiding the overlay.

**Comment by Prefix1802** [comment]
@nnixaa 

Nothing i can think of right now, but when ithink of something i will let you know ;)

**Comment by nnixaa** [comment]
@Prefix1802 cool then! Hope there is no blocker for particularly your or anyone else's project :)

**Comment by bekim-bajraktari** [comment]
Hi together,
can someone tell me where can I fix that issues with sidebar-width-compact?
Can I fix that directly in file sidebar.component.css?
